src/test/developTest/java/mServer/crawler/sender/newsearch/ZdfIndexPageDeserializerTest.java

```diff
@@ -0,0 +1,51 @@
+package mServer.crawler.sender.newsearch;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)